src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -138,6 +139,14 @@ def disk_items(disk)
         CWM::PagerTreeItem.new(page, children: children)
       end
 
+      # Hard disks section contains disks, dasd and multipath devices
+      #
+      # @return [Array<Y2Storage::BlkDevice>]
+      def disk_devices
+        disk_types = [Y2Storage::Disk, Y2Storage::Dasd, Y2Storage::Multipath]
+        device_graph.disk_devices.select { |d| disk_types.include?(d.class) }
```

Comment:
Wouldn't this be nicer (more duck-typed and more robust for inheritance) like?
``ruby
device_graph.disk_devices.select { |d| d.is?(:disk, :dasd, :multipath) }
```

Reply:
That's exactly what `is?` was created for http://www.rubydoc.info/github/yast/yast-storage-ng/master/Y2Storage/Device#is%3F-instance_method

Reply:
Of course, I did not remember the multiparams ability  of `#is?`.